packages/neuron-wallet/src/models/multi-sign-utils.ts

```diff
@@ -0,0 +1,53 @@
+import { SystemScript } from "./lock-utils"
+import NodeService from "services/node"
+import OutPoint from "./chain/out-point"
+import { ScriptHashType } from "./chain/script"
+import RpcService from "services/rpc-service"
+
+export default class MultiSignUtils {
+  multiSignScript: SystemScript
+
+  constructor(multiSignScript: SystemScript) {
+    this.multiSignScript = multiSignScript
+  }
+
+  private static multiSignScriptInfo: SystemScript | undefined
+
+  private static previousURL: string | undefined
+
+  static async loadMultiSignScript(nodeURL: string): Promise<SystemScript> {
+    const rpcService = new RpcService(nodeURL)
+
+    const genesisBlock = await rpcService.getBlockByNumber('0')
+    const tx = genesisBlock!.transactions[1]
+    const outPoint = new OutPoint(tx.hash!, '1')
+    const codeHash = tx.outputs[4].type!.computeHash()
+    const hashType = ScriptHashType.Type
+
+    return {
+      codeHash,
+      outPoint,
+      hashType,
+    }
+  }
+
+  static async multiSignScript(nodeURL: string = NodeService.getInstance().ckb.rpc.node.url): Promise<SystemScript> {
+    if (MultiSignUtils.multiSignScriptInfo && nodeURL === MultiSignUtils.previousURL) {
```

Comment:
Is a comparison between genesis block hashes better than that between URLs?

Reply:
Can't read from genesis block, especially off network.
And no need to update this alone, should refactor with `LockUtils` and `DaoUtils` and merge.